Index: openssl-1.0.1/apps/s_client.c
===================================================================
--- openssl-1.0.1.orig/apps/s_client.c	2012-03-14 14:32:14.000000000 +0100
+++ openssl-1.0.1/apps/s_client.c	2012-03-18 16:08:29.000000000 +0100
@@ -622,13 +622,7 @@
 	SRP_ARG srp_arg = {NULL,NULL,0,0,0,1024};
 #endif
 
-#if !defined(OPENSSL_NO_SSL2) && !defined(OPENSSL_NO_SSL3)
 	meth=SSLv23_client_method();
-#elif !defined(OPENSSL_NO_SSL3)
-	meth=SSLv3_client_method();
-#elif !defined(OPENSSL_NO_SSL2)
-	meth=SSLv2_client_method();
-#endif
 
 	apps_startup();
 	c_Pause=0;
Index: openssl-1.0.1/apps/s_server.c
===================================================================
--- openssl-1.0.1.orig/apps/s_server.c	2012-02-12 00:21:09.000000000 +0100
+++ openssl-1.0.1/apps/s_server.c	2012-03-18 16:09:21.000000000 +0100
@@ -969,17 +969,7 @@
 	char *srpuserseed = NULL;
 	char *srp_verifier_file = NULL;
 #endif
-#if !defined(OPENSSL_NO_SSL2) && !defined(OPENSSL_NO_SSL3)
 	meth=SSLv23_server_method();
-#elif !defined(OPENSSL_NO_SSL3)
-	meth=SSLv3_server_method();
-#elif !defined(OPENSSL_NO_SSL2)
-	meth=SSLv2_server_method();
-#elif !defined(OPENSSL_NO_TLS1)
-	meth=TLSv1_server_method();
-#else
-  /*  #error no SSL version enabled */
-#endif
 
 	local_argc=argc;
 	local_argv=argv;
